sabnzbd/filesystem.py

```diff
@@ -300,16 +300,16 @@ def create_real_path(name, loc, path, umask=False, writable=True):
     if path:
         my_dir = real_path(loc, path)
         if not os.path.exists(my_dir):
-            logging.info("%s directory: %s does not exist, try to create it", name, my_dir)
+            logging.info("{} directory: {} does not exist, try to create it".format(name, my_dir))
```

Comment:
I enjoy the new `.format()`, but in the case of logging is it really more readable than using the parameters to the `info`/`debug`/etc functions?
That's also the way how we do it in all the other files :)

Reply:
You are absolutely right, so I removed the 'format' from all logging functions.  As you stated elsewhere, I left it in for the string modifications.  It's a bit more robust when handling integers, etc., so there is some value in using it.